src/Apps/Order/Routes/Review/index.tsx

```diff
@@ -41,10 +41,30 @@ export interface ReviewProps {
   isCommittingMutation: boolean
 }
 
+export interface ReviewState {
+  stripe: stripe.Stripe
+}
+
 const logger = createLogger("Order/Routes/Review/index.tsx")
 
 @track()
-export class ReviewRoute extends Component<ReviewProps> {
+export class ReviewRoute extends Component<ReviewProps, ReviewState> {
+  state = { stripe: null }
+  componentDidMount() {
+    if (window.Stripe) {
+      this.setState({
+        stripe: window.Stripe(window.sd.STRIPE_PUBLISHABLE_KEY),
```

Comment:
@lilyfromseattle we talked about this, @ds300 @sweir27 is there a better way to get access to current stripe context and not get it from `window`?

Reply:
Usually we import `sd` variables through the `"sharify"` module. So add this:
```
import { data as sd } from "sharify"
```
to your imports and you can pull it directly off of `sd`. ([Example here](https://github.com/artsy/reaction/blob/master/src/Components/Payment/PaymentFormWrapper.tsx#L4))